src/screens/GroupInfo/GroupMedia.js

```diff
@@ -79,12 +79,12 @@ const GroupMedia = () => {
 export default GroupMedia;
 
 const styles = StyleSheet.create({
-  container: {flex: 1, backgroundColor: '#fff'},
+  container: {flex: 1, backgroundColor: COLORS.white},
 
   toptabcontainer: {
     flexDirection: 'row',
-    backgroundColor: colors.white,
-    borderBottomColor: '#00000050',
+    backgroundColor: COLORS.white,
+    borderBottomColor: COLORS.black30percent,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Replacing hard-coded color values with references to color constants from a theme file improves maintainability. If the color scheme needs to be updated in the future, changes will only need to be made in one place.
```diff
- container: {flex: 1, backgroundColor: '#fff'},
- toptabcontainer: {
-   flexDirection: 'row',
-   backgroundColor: colors.white,
-   borderBottomColor: '#00000050',
-   borderBottomWidth: 1,
-   paddingHorizontal: 4
- },
+ container: {flex: 1, backgroundColor: COLORS.white},
+ toptabcontainer: {
+   flexDirection: 'row',
+   backgroundColor: COLORS.white,
+   borderBottomColor: COLORS.black30percent,
+   borderBottomWidth: 1,
+   paddingHorizontal: 4
+ },
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->